<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtGraphs</class>
 <widget class="QMainWindow" name="QtGraphs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtGraphs</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_AddEdge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить грань</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_SecondNode">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleteNode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить узел</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_DeleteEdge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить грань</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_AddNode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>добавить узел</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_addDelNodeValue">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_UpdateEdge">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить грань</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LineEdit_FirstNode">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Weight">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>170</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_CLEAR">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>510</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>очистка</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ZagotovlenGraph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Заготовленный граф</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_randGraph">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Случайный граф</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_KomiVoyager">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Задача Комивояжера</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_KomiVoyager">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>410</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_KomiVoyager">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1071</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
